master/internal/rm/kubernetesrm/pod.go

```diff
@@ -74,12 +75,13 @@ type pod struct {
 	// TODO: Drop this manufactured container obj all together.
 	container        cproto.Container
 	ports            []int
-	resourcesDeleted bool
+	resourcesDeleted atomic.Bool
 	containerNames   set.Set[string]
 
-	logCtx logger.Context
-
 	restore bool
+
+	syslog           *logrus.Entry
+	resourceErrorCtx func(ctx *actor.Context) errorCallbackFunc
```

Comment:
this name isn't meaningful to me

Reply:
hmmm, is `handleResourceError` any better? would you happen to have any suggestions?